controller/volume_controller.go

```diff
@@ -452,13 +452,46 @@ func (vc *VolumeController) syncVolume(key string) (err error) {
 		return err
 	}
 
+	if err := vc.reuseFailedUpgradingReplica(volume, replicas); err != nil {
```

Comment:
It's better not to add an extra replica scheduling-related function. It makes the scheduling flow inside volume-controller become more complicated...
For the **failed** new replicas during the upgrade, I am fine with directly cleaning them up then starting new replicas...

Reply:
OK, Got it.
Removed, and try to postpone the replica launching.